kraken_frontend/src/svg/users.tsx

```diff
@@ -14,80 +14,80 @@ export default function UsersIcon(props: React.HTMLAttributes<HTMLDivElement>) {
                         />
                         <path
                             d="M245.1,155.558h66.5c-0.4-27.7,0.8-42.5-17.2-49c-16.3-6.2-26.1-12.6-26.1-12.6l-12.6,39.7l-1.7,5.4l-5.6-15.9
-			c12.9-18-1-18.9-3.4-18.9l0,0l0,0l0,0l0,0l0,0l0,0c-2.4,0-16.3,0.9-3.4,18.9l-5.6,15.9l-1.7-5.4l-12.6-39.7c0,0-9.8,6.4-26.1,12.6
-			c-18,6.5-16.8,21.3-17.2,49H245H245.1z"
+                            c12.9-18-1-18.9-3.4-18.9l0,0l0,0l0,0l0,0l0,0l0,0c-2.4,0-16.3,0.9-3.4,18.9l-5.6,15.9l-1.7-5.4l-12.6-39.7c0,0-9.8,6.4-26.1,12.6
+                            c-18,6.5-16.8,21.3-17.2,49H245H245.1z"
                         />
                         <path
                             d="M216.8,68.758c1.4,9.3,8.6,21.2,20.5,25.4c4.8,1.7,10.2,1.7,15,0c11.6-4.2,19.1-16,20.5-25.3c1.6-0.1,3.6-2.3,5.8-10.2
-			c3-10.7-0.2-12.3-2.9-12.1c0.5-1.5,0.9-3,1.2-4.4c4.6-27.8-9.1-28.8-9.1-28.8s-2.3-4.4-8.2-7.7c-4-2.4-9.6-4.2-16.9-3.6
-			c-2.4,0.1-4.6,0.6-6.7,1.3l0,0c-2.7,0.9-5.2,2.2-7.4,3.8c-2.8,1.7-5.3,3.9-7.6,6.3c-3.6,3.7-6.9,8.5-8.3,14.5
-			c-1.2,4.5-0.9,9.1,0.1,14.1l0,0c0.3,1.5,0.7,2.9,1.2,4.4c-2.7-0.3-6,1.3-2.9,12.1C213.2,66.458,215.2,68.658,216.8,68.758z"
+                            c3-10.7-0.2-12.3-2.9-12.1c0.5-1.5,0.9-3,1.2-4.4c4.6-27.8-9.1-28.8-9.1-28.8s-2.3-4.4-8.2-7.7c-4-2.4-9.6-4.2-16.9-3.6
+                            c-2.4,0.1-4.6,0.6-6.7,1.3l0,0c-2.7,0.9-5.2,2.2-7.4,3.8c-2.8,1.7-5.3,3.9-7.6,6.3c-3.6,3.7-6.9,8.5-8.3,14.5
+                            c-1.2,4.5-0.9,9.1,0.1,14.1l0,0c0.3,1.5,0.7,2.9,1.2,4.4c-2.7-0.3-6,1.3-2.9,12.1C213.2,66.458,215.2,68.658,216.8,68.758z"
                         />
                         <path
                             d="M300.7,248.158c-30.7-30.7-80.6-30.7-111.3,0c-29.6,29.6-30.7,76.9-3.3,107.8l0.8,0.7c-3,6.4-8.1,14.3-16.2,18.3
-			c-2.7,1.3-2.1,5.3,0.9,5.8c8.9,1.4,21.6-0.2,33.5-9.1l0.2,0.1c30.2,17.7,69.6,13.7,95.5-12.2
-			C331.4,328.758,331.4,278.858,300.7,248.158z M211.1,313.958c-5.6,0-10.2-4.6-10.2-10.2s4.6-10.2,10.2-10.2
-			c5.6,0,10.2,4.6,10.2,10.2S216.8,313.958,211.1,313.958z M245,313.958c-5.6,0-10.2-4.6-10.2-10.2s4.6-10.2,10.2-10.2
-			c5.6,0,10.2,4.6,10.2,10.2S250.7,313.958,245,313.958z M278.9,313.958c-5.6,0-10.2-4.6-10.2-10.2s4.6-10.2,10.2-10.2
-			s10.2,4.6,10.2,10.2S284.6,313.958,278.9,313.958z"
+                            c-2.7,1.3-2.1,5.3,0.9,5.8c8.9,1.4,21.6-0.2,33.5-9.1l0.2,0.1c30.2,17.7,69.6,13.7,95.5-12.2
+                            C331.4,328.758,331.4,278.858,300.7,248.158z M211.1,313.958c-5.6,0-10.2-4.6-10.2-10.2s4.6-10.2,10.2-10.2
+                            c5.6,0,10.2,4.6,10.2,10.2S216.8,313.958,211.1,313.958z M245,313.958c-5.6,0-10.2-4.6-10.2-10.2s4.6-10.2,10.2-10.2
+                            c5.6,0,10.2,4.6,10.2,10.2S250.7,313.958,245,313.958z M278.9,313.958c-5.6,0-10.2-4.6-10.2-10.2s4.6-10.2,10.2-10.2
+                            s10.2,4.6,10.2,10.2S284.6,313.958,278.9,313.958z"
                         />
                     </g>
```

Comment:
Run prettier

Reply:
this is already prettified, that's why it's using spaces instead of tabs now